frontend/src/metabase/visualizations/echarts/cartesian/model/index.ts

```diff
@@ -97,8 +97,13 @@ export const getCartesianChartModel = (
   }
   dataset = sortDataset(dataset, settings["graph.x_axis.scale"]);
 
+  const xAxisSeriesModel = seriesModels.find(
```

Comment:
I think this will always be undefined. Series represent lines/bars/areas on a chart so there is no series with only x-axis values and there is no way to specify a name for it. But to understand what dimension x-axis uses there is dimensionModel. So probably we don't need it but please let me know if you were solving some specific case with it

Reply:
Yeah, wasn't sure if that makes sense at all, I'll remove the X axis part. Thanks!